tests/pixie/tests/test-stdlib.pxi

```diff
@@ -282,4 +282,11 @@
   (try
      (throw "This is an exception")
      (catch e
-       (t/assert= (ex-msg e) "This is an exception"))))
\ No newline at end of file
+       (t/assert= (ex-msg e) "This is an exception"))))
+
+(t/deftest test-range
+  (t/assert= (-seq (range 10))
+             (-seq (-iterator (range 10))
+             (reduce conj nil (range 10))
+             '(0 1 2 3 4 5 6 7 8 9))))
```

Comment:
Nice catch! It makes sense that it only takes 2 arguments I will just wrap the three statements in (= ...) and make sure that it returns true.

I will add another pull request which makes assert= throw an error when not given 2 arguments.